### PR TITLE
updated version to 15.3.0

### PR DESCRIPTION
**Pull request by jennahamlin**
# Submitting a Container
 
 (If you're requesting for a new container, please check the procedure described [here](https://github.com/BioContainers/containers#241-how-to-request-a-container).  
 
 ## Check BioContainers' Dockerfile [specifications](https://github.com/BioContainers/specs)  
 ## Checklist
 
 1. Misc
 - [x] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [x] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [x] LABEL base_image  
 - [x] LABEL version
 - [x] LABEL software.version  
 - [x] LABEL about.summary  
 - [x] LABEL about.home  
 - [x] LABEL about.license   
 - [x] MAINTAINER  
 
 3. Extra (optionals)
  - [ ] I have written tests in test-cmds.txt  
  - [ ] LABEL extra.identifier    
  - [ ] LABEL about.documentation   
  - [ ] LABEL about.license_file
  - [ ] LABEL about.tags   
 
 
 ## Check [BioContainers'](https://github.com/BioContainers/specs) Dockerfile metadata

### CODE REVIEW

File: ncbi-datasets-cli/15.3.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+FROM biocontainers/biocontainers:v1.2.0_cv2
+
+################## METADATA ######################
+
+LABEL base_image="biocontainers:v1.2.0_cv2"
+LABEL version="1"
+LABEL about.summary="NCBI Datasets is a new resource that lets you easily gather data from across NCBI databases."
+LABEL software="ncbi-datasets-cli"
+LABEL software.version="14.2.2"
```
Comment (osallou) [inline ncbi-datasets-cli/15.3.0/Dockerfile]: you forgot to updated related version metadata

**Review by osallou — CHANGES_REQUESTED** [review]

**Review by jennahamlin — COMMENTED** [review]
MetaData changes: changed label version 2 -> 3, unsure if this is necessary. Changed label software.version 14.2.2 -> 15.3.0

**Comment by osallou** [comment]
Buld failed on conda install, did you try locally the docker build?

**Comment by jennahamlin** [comment]
Hmm, interesting that the build for 15.3.0 failed. I tried to locally build via docker build for both 15.3.0 and 14.2.2 and both failed at the same step (fetching package metadata) and with same exit code (137). This maybe a certificate issue or a me issue, as I am probably missing something.

**Comment by osallou** [comment]
seems related to old conda version used in image, does not work anymore (at least not for all)

a conda update conda fix the issue

    RUN conda update -n base conda
    RUN conda install ...

**Comment by jennahamlin** [comment]
Thank you for the comments but found the container: quay.io/staphb/ncbi-datasets:15.2.0
No need to duplicate

**Pull request closed without merging**